assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -125,7 +125,7 @@ class TextBlockEdit extends Component {
 					onChange={ ( nextContent ) => setAttributes( { content: nextContent } ) }
 					// The 2 following lines are necessary for pasting to work.
 					onReplace={ this.onReplace }
-					onSplit={ () => {} }
+					unstableOnSplit={ () => {} }
```

Comment:
This is replaced because `onSplit` will be deprecated soon.

Reply:
Where can I learn more about this being deprecated soon?

Reply:
Here is the notice: https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/rich-text/index.js#L117-L126

However, just saw there is a [PR to address this](https://github.com/WordPress/gutenberg/pull/14765), so, perhaps it's better to keep it `onSplit`.